build.gradle

```diff
@@ -21,8 +21,8 @@ buildscript {
         jcenter()
     }
     dependencies {
-        // NOTE: Do not place your application dependencies here; they belong
-        // in the individual module build.gradle files
+        classpath 'com.android.tools.build:gradle:2.3.1'
```

Comment:
Is this still required for a Java only project?

Reply:
Good question, file an issue or send a PR for this to discuss separately, I'm not sure.